bindata/network/multus/multus.yaml

```diff
@@ -436,6 +436,7 @@ spec:
               "kubernetes": {
                 "kubeconfig": "/etc/kubernetes/cni/net.d/whereabouts.d/whereabouts.kubeconfig"
               },
+              "reconciler_cron_expression": "30 4 * * *",
```

Comment:
This looks like we're running reconciler once a day at 4:30am - is this what we want?

Reply:
Thanks for the check -- yeah, I think once a day for the like "audit style reconciliation" is good. Happy to take other input, too